### DISCUSSION TITLE
Create a step-line chart

### DISCUSSION THREAD

**Discussion by lorisd38** [eclipse-birt/birt#1541] [Q&A]
Hello,

I'm new to BIRT and I'm currently trying to build a step-line graph. The only solution I've found is to use scripts but I don't know how to do it and I can't find any information on them.

I try something but it doesn't work cause chart.getDataSet() doesn't exist...
![image](https://github.com/eclipse-birt/birt/assets/33525019/79a7e8c9-d05f-48d6-afba-dd041a558a7d)

And I'm using BIRT Report Designer RCP 4.13.0.

Thank you for your help.

**Comment by lorisd38** [comment]
I find how to do it.

**Comment by raviprsd18** [comment]
@lorisd38 can you please share the working code here. I too have the same requirement.
